Applications/VpView/vpVidtkTrackIO.cxx

```diff
@@ -47,6 +48,72 @@ vpVidtkTrackIO::~vpVidtkTrackIO()
 
 //-----------------------------------------------------------------------------
 bool vpVidtkTrackIO::ReadTracks()
+{
+  return this->ReadTracks(0);
+}
+
+//-----------------------------------------------------------------------------
+vtkIdType vpVidtkTrackIO::ComputeNumberOfPoints(
+  const vpFileTrackIOImpl::TrackRegionMapType* trackRegionMap)
```

Comment:
(Note for the record: that *would* have been the case, except I wound up writing the `getValue` helpers to take a pointer to the container and check if it is null, which allowed the same simplification without passing a never-null but possibly-empty map. Particularly, this was done because one spot needs to do a two-tiered look-up, and would otherwise have still needed a separate check for null, so the overload taking a pointer-to-container was desired anyway.)